liquidator/test/index.js

```diff
@@ -88,15 +88,15 @@ contract("index.js", function(accounts) {
   it("Sets token allowances correctly", async function() {
     const priceFeedConfig = defaultPricefeedConfig;
 
-    await Poll.run(emp.address, false, 10_000, priceFeedConfig);
+    await Poll.run(emp.address, 0, priceFeedConfig);
 
     const collateralAllowance = await collateralToken.allowance(contractCreator, emp.address);
     assert.equal(collateralAllowance.toString(), MAX_UINT_VAL);
     const syntheticAllowance = await syntheticToken.allowance(contractCreator, emp.address);
     assert.equal(syntheticAllowance.toString(), MAX_UINT_VAL);
   });
-
-  it("Responds to incoming monitor requests while bot is alive", async function() {
+  // TODO: remove this test when we remove the black box testing URL.
+  it.skip("Responds to incoming monitor requests while bot is alive", async function() {
```

Comment:
This test should be removed and for now it'll get skipped.

Reply:
You should merge master and remove this